src/blocks/progress-bar/edit.js

```diff
@@ -0,0 +1,57 @@
+
+import classname from 'classnames';
+
+
+const { __ } = wp.i18n;
+const { RichText } = wp.blockEditor;
+const { Fragment } = wp.element;
+
+import Inspector from './inspector.js';
+
+const ProgressBar = ({ attributes, setAttributes }) => {
+
+	const { text, value, progressColor, backgroundColor, borderRadius, height, animated, hideValue } = attributes;
```

Comment:
We shouldn't destructure them as it makes it difficult to know which one is attribute.